frontend/pages/DashboardPage/cards/ActivityFeed/ActivityItem/ActivityItem.tests.tsx

```diff
@@ -259,6 +259,15 @@ describe("Activity Feed", () => {
       screen.getByText("successfully logged in from public IP 192.168.0.1.")
     ).toBeInTheDocument();
   });
+  it("renders a user_logged_in type activity without public IP", () => {
+    const activity = createMockActivity({
+      type: ActivityType.UserLoggedIn,
+      details: {},
```

Comment:
Originally put `public_ip: undefined` but it looks like other tests use empty object `{}` and I don't think someone will add `public_ip` to the mock activity